TTT.h

```diff
@@ -0,0 +1,10 @@
+#include "pole.h"
+
+class TTT 
+{
+	public:
```

Comment:
I suppose this means we've made a conscious decision to use tabs over spaces? 😁 I'm all for it, but we should set our editors appropriately with the proper tabstop if this is the case.